app/ts/utils/requests.ts

```diff
@@ -53,3 +53,11 @@ export const getUniqueRequestIdentifierString = (uniqueRequestIdentifier: Unique
 export const doesUniqueRequestIdentifiersMatch = (a: UniqueRequestIdentifier, b: UniqueRequestIdentifier) => {
 	return a.requestId == b.requestId && a.requestSocket.connectionName === b.requestSocket.connectionName && a.requestSocket.tabId === b.requestSocket.tabId
 }
+
+export async function fetchWithTimeout(resource: RequestInfo | URL, init?: RequestInit | undefined, timeoutS: number = 60000) {
```

Comment:
`timeoutS` may be a typo here with a capital `S`. or you might be referring to milliseconds which in that case could have been `timeoutms`